x/distribution/legacy/v0_36/migrate.go

```diff
@@ -0,0 +1,29 @@
+package v0_36
+
+import (
+	v034distr "github.com/cosmos/cosmos-sdk/x/distribution/legacy/v0_34"
+)
+
+func Migrate(oldGenState v034distr.GenesisState) GenesisState {
+	// migrate slash events which now have the period included
+	slashEvents := make([]ValidatorSlashEventRecord, len(oldGenState.ValidatorSlashEvents), len(oldGenState.ValidatorSlashEvents))
+	for i, se := range oldGenState.ValidatorSlashEvents {
+		slashEvents[i] = ValidatorSlashEventRecord{
+			ValidatorAddress: se.ValidatorAddress,
+			Height:           se.Height,
+			Period:           se.Event.ValidatorPeriod,
```

Comment:
Please confirm this is correct @rigelrozanski.

Reply:
This is correct